models/machine.py

```diff
@@ -57,6 +64,20 @@ def run_filters_and_skip_reason(self, benchmarkable_type, override_filters=None)
 
         machine_run_filters = deepcopy(self.default_filters[benchmarkable_type])
 
+        if (
+            "commit_message_skip_strings" in machine_run_filters
+            # never skip PRs or wheels with otherwise-skippable commit messages
+            and benchmarkable_reason.endswith("-commit")
```

Comment:
nit: listing possible `benchmarkable_reason` values in the comment (or pointing to somewhere where they're listed) would make the cases here easier to understand

Reply:
Cool, I'll add more comments here